src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs

```diff
@@ -43,7 +43,7 @@ public ActiveDirectoryConfiguration ActiveDirectoryConfiguration
         private string _roleAdminGroupDN;
         
         /// <summary>
-        /// Optional. Gets or sets the Distinguished Name for role
+        /// Optional. Optional. Gets or sets the Distinguished Name for role
```

Comment:
@lyerrb Yes, this is because the geenrator adds the oiptional keyword.  Please remove the extra Optional